ffi/SDL2_0.lua

```diff
@@ -21,6 +21,8 @@ require("ffi/linux_input_h")
 -----------------------------------------------------------------
 
 local SDL = util.ffiLoadCandidates{
+    "/usr/local/opt/lib/libSDL2.dylib",
+    "/opt/homebrew/lib/libSDL2.dylib",
```

Comment:
This should go lower since SDL2 is the default. See <http://luajit.org/ext_ffi_api.html#ffi_load>.

>  On POSIX systems, if the name contains no dot, the extension .so is appended. Also, the lib prefix is prepended if necessary. So ffi.load("z") looks for "libz.so" in the default shared library search path.
>
> On Windows systems, if the name contains no dot, the extension .dll is appended. So ffi.load("ws2_32") looks for "ws2_32.dll" in the default DLL search path.